examples/AspNet/Controllers/WeatherForecastController.cs

```diff
@@ -57,7 +57,7 @@ public async Task<IEnumerable<WeatherForecast>> Get(int customerId)
     {
         if (customerId < 0)
         {
-            throw new ArgumentException();
+            throw new ArgumentOutOfRangeException(nameof(customerId));
```

Comment:
nit: Add a message saying what's the expected range for customerId

Reply:
Updated